ci_framework/roles/tempest/tasks/configure-tempest.yml

```diff
@@ -27,3 +27,16 @@
   ansible.builtin.template:
     src: clouds.yaml.j2
     dest: "{{ cifmw_tempest_artifacts_basedir }}/clouds.yaml"
+
+# As an example:
+# cifmw_tempest_tempestconf_profile:
+#   overrides:
+#     validation.run_ssh: false
+#     telemetry.alarm_granularity: '60'
+
+- name: Create profile.yaml file
+  when: cifmw_tempest_tempestconf_profile
```

Comment:
we are hitting following error [here](https://logserver.rdoproject.org/07/407/35f97a458fa439637602a92ff973f542733f1d90/github-check/cifmw-end-to-end-nobuild-tagged/7b80ddc/job-output.txt)
```
TASK [tempest : Create profile.yaml file content={{ cifmw_tempest_tempestconf_profile | to_nice_yaml }}, dest={{ cifmw_tempest_artifacts_basedir }}/profile.yaml, mode=0644] ***
2023-07-27 00:46:44.232441 | controller | Thursday 27 July 2023  00:46:44 -0400 (0:00:00.553)       0:20:42.537 *********
2023-07-27 00:46:44.279305 | controller | fatal: [localhost]: FAILED! => {"msg": "The conditional check 'cifmw_tempest_tempestconf_profile' failed. The error was: error while evaluating conditional (cifmw_tempest_tempestconf_profile): 'cifmw_tempest_tempestconf_profile' is undefined. 'cifmw_tempest_tempestconf_profile' is undefined\n\nThe error appears to be in '/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles/tempest/tasks/configure-tempest.yml': line 37, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Create profile.yaml file\n  ^ here\n"}
```
We can change when condition to
```
when: cifmw_tempest_tempestconf_profile is defined
```